template-skeleton/template/package.dev.json

```diff
@@ -43,13 +44,13 @@
     "eslint-config-prettier": "^7.2.0",
     "eslint-plugin-prettier": "^3.3.1",
     "firebase-tools": "^9.2.2",
-    "start-server-and-test": "^1.12.0",
     "husky": "^4.3.8",
     "hygen": "^6.1.0",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1",
     "react-app-rewired": "^2.1.8",
-    "react-is": "^17.0.1"
+    "react-is": "^17.0.1",
+    "start-server-and-test": "^1.12.0"
```

Comment:
Why this?

Reply:
`start-server-and-test` is already installed from before

Reply:
What do you mean `from before`?

Reply:
In `npm install` the package.json is restructured single. This dependencie was before

Reply:
Got it